### PR TITLE
Form designed in html

### PR DESCRIPTION
**Pull request by LottieRugg**

### CODE REVIEW

**Review by alimcgarry — COMMENTED** [review]
Yup, all looks good to me! 

Big Al

File: data.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+session_start();
```
Comment (ashleycoles) [inline data.php]: https://media.giphy.com/media/s239QJIh56sRW/giphy.gif

File: data.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+require 'functions.php';
+$db = connectToDatabase();
+
+$name_input= $_POST["name"];
+$address_input= $_POST["address"];
+$rating_input= $_POST["rating"];
+$local_brewery_input= $_POST["local_brewery"];
+$picture_input= $_POST["picture"];
+$what_to_order_input= $_POST["what_to_order"];
+$open_fire_input= $_POST["open_fire"];
```
Comment (ashleycoles) [inline data.php]: Definitely worth wrapping these in an isset to make sure some naughty user hasn't directly navigated to data.php

`if (isset($_POST['name']) {` will do the trick, no need to run all the POST items through isset

File: data.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+$query->bindParam(':name',$name_input);
+$query->bindParam(':address', $address_input);
+$query->bindParam(':rating', $rating_input);
+$query->bindParam(':local_brewery', $local_brewery_input);
+$query->bindParam(':picture', $picture_input);
+$query->bindParam(':what_to_order', $what_to_order_input);
+$query->bindParam(':open_fire', $open_fire_input);
+
+$query-> execute();
+?>
```
Comment (ashleycoles) [inline data.php]: You might want to redirect the user back to the pubs_of_sheffield.php page at this point

File: data.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+<html>
+<head>
+
+</head>
+
+</html>
```
Comment (ashleycoles) [inline data.php]: If you redirect you can ditch this HTML - this data.php file just acts a middle man to capture and save the data before sending the user back home

File: pubs_of_sheffield.php [hunk 5]
```diff
@@ -2,17 +2,16 @@
 require 'functions.php';
 $db = connectToDatabase();
 $itemsList = getAllItems($db);
+session_start()
```
Comment (ashleycoles) [inline pubs_of_sheffield.php]: You can axe this, session isn't needed to use forms, only when you want to start tracking info about a user like whether or not they are logged in

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: data.php (hunk 1 above)
Comment (LottieRugg) [reply data.php]: hahaha

Comment (LottieRugg) [reply data.php]: fresh sesh

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**